oss-internship-2020/libtiff/test/helper.cc

```diff
@@ -18,20 +18,24 @@
 #include "sandboxed_api/sandbox2/util/fileops.h"
 #include "sandboxed_api/sandbox2/util/path.h"
 
-std::string GetImagesFolder() {
+std::string GetFilePath(const absl::string_view filename) {
```

Comment:
An `absl::string_view` is an immutable view of a string. So `const` is neither required nor desired here.